lib/active_merchant/billing/gateways/airwallex.rb

```diff
@@ -0,0 +1,249 @@
+module ActiveMerchant #:nodoc:
+  module Billing #:nodoc:
+    class AirwallexGateway < Gateway
+      self.test_url = 'https://api-demo.airwallex.com/api/v1'
+      self.live_url = 'https://pci-api.airwallex.com/api/v1'
+
+      # per https://www.airwallex.com/docs/online-payments__overview, cards are accepted in all EU countries
+      self.supported_countries = %w[AT AU BE BG CY CZ DE DK EE GR ES FI FR GB HK HR HU IE IT LT LU LV MT NL PL PT RO SE SG SI SK]
+      self.default_currency = 'AUD'
+      self.supported_cardtypes = %i[visa master]
+
+      self.homepage_url = 'https://airwallex.com/'
+      self.display_name = 'Airwallex'
+
+      ENDPOINTS = {
+        login: '/authentication/login',
+        setup: '/pa/payment_intents/create',
+        sale: '/pa/payment_intents/%{id}/confirm',
+        capture: '/pa/payment_intents/%{id}/capture',
```

Comment:
Nice, I know about string interpolation in Python but I didn't realize this was supported in Ruby